### PR TITLE
Don't use Get-Member to get vsvwhere version.

### PR DESCRIPTION
**Pull request by mmitche**
Get-Member returns the whole json node rather than the value, as intended.

### CODE REVIEW

**Comment by mmitche** [comment]
@tmat You changed this to use Get-Member, which could have never worked based on how this was specified in the global.json.

**Comment by tmat** [comment]
Ah, yes, I corrected this mistake in a couple of places and this one fell through. 

Reading directly is incorrect as well though since it throws if the node isn't there. 
The correct code is 

```
  if (Get-Member -InputObject $GlobalJson.tools -Name "vswhere") {	
    $vswhereVersion = $GlobalJson.tools.vswhere
  } else {
    $vswhereVersion = "2.5.2"
  }
```

File: eng/common/tools.ps1 [hunk 1]
```diff
@@ -176,8 +176,9 @@ function InstallXCopyMSBuild([string] $packageVersion) {
 # or $null if no instance meeting the requirements is found on the machine.
 #
 function LocateVisualStudio {
-  $vswhereVersion = $GlobalJson.tools.vswhere
-  if ($vsWhereVersion -eq $null) {
+  if ($vswhereVersion = Get-Member -InputObject $GlobalJson.tools -Name "vswhere") {
```
Comment (tmat) [inline eng/common/tools.ps1]: `$vswhereVersion = ` is unnecessary in the if condition

**Comment by tmat** [comment]
Thanks for fixing this!

**Pull request merged**